### PR TITLE
Form close detection

### PR DESCRIPTION
**Pull request by Endermanbugzjfc**
Code has shitty doc please fix doc

### CODE REVIEW

File: server/player/form/submit.go [hunk 1]
```diff
@@ -30,6 +30,12 @@ type MenuSubmittable interface {
 // buttons on a Modal form will not have images.
 type ModalSubmittable MenuSubmittable
 
+// Closer represents a form which has special logic when being closed by a Submitter. The struct struct will have its Close method called, when its form is closed.
```
Comment (Lucaskyy) [inline server/player/form/submit.go:33]: `struct struct` -> `form`
`its form` -> `it is`

**Review by Lucaskyy — CHANGES_REQUESTED** [review]

**Comment by Lucaskyy** [comment]
Isn't `Closable` a better name than `Closer`?

File: server/player/form/submit.go [hunk 2]
```diff
@@ -30,6 +30,12 @@ type MenuSubmittable interface {
 // buttons on a Modal form will not have images.
 type ModalSubmittable MenuSubmittable
 
+// Closer represents a form which has special logic when being closed by a Submitter. The form will have its Close method called, when it is closed.
```
Comment (Sandertv) [inline server/player/form/submit.go:33]: Would remove the `The form...is closed' sentence as the Close method documents that.

File: server/player/form/submit.go [hunk 3]
```diff
@@ -30,6 +30,12 @@ type MenuSubmittable interface {
...
+type Closer interface {
```
Comment (Sandertv) [inline server/player/form/submit.go:34]: Please add a `.` at the end.

**Review by Sandertv — COMMENTED** [review]
Just some documentation nitpicks, looks good otherwise.

**Review by Lucaskyy — APPROVED** [review]

**Review by Sandertv — APPROVED** [review]
Looks good, thank you!

**Pull request merged**